pom.xml

```diff
@@ -1,57 +1,85 @@
+        <relativePath />
+    </parent>
+    <groupId>io.jenkins.plugins</groupId>
+    <artifactId>copy-to-slave</artifactId>
+    <version>1.4.8</version>
+    <packaging>hpi</packaging>
+    <properties>
+        <jenkins.version>1.625</jenkins.version>
+        <java.level>8</java.level>
+        <!-- TODO: JENKINS-36716 - Switch to Medium once FindBugs is cleaned up -->
+        <!-- <findbugs.effort>Max</findbugs.effort>
+        <findbugs.threshold>High</findbugs.threshold> -->
+        <findbugs.excludeFilterFile>${project.basedir}/src/findbugs/findbugs-excludes.xml</findbugs.excludeFilterFile>
+    </properties>
+    <name>Copy To Slave Plugin</name>
+    <description>Copy files between the master and agent (Insecure)</description>
+    <url>http://wiki.jenkins-ci.org/display/JENKINS/Copy+To+Slave+Plugin</url>
+    <developers>
+        <developer>
+            <id>jmcgeheeiv</id>
+            <name>John McGehee</name>
+            <email>jmcgeheeiv@users.noreply.github.com</email>
+        </developer>
+        <developer>
+            <id>svvivek</id>
+            <name>Vivekanand S V</name>
+            <email>svvivekanand@gmail.com</email>
+            <timezone>+5.5</timezone>
+         </developer>
 	 <developer>
-	 	<id>sshelomentsev</id>
-		<name>Sergey Shelomentsev</name>
-		<email>sergey.v.shelomentsev@gmail.com</email>
-		<timezone>+7</timezone>
-	 </developer>
-  </developers>
+             <id>sshelomentsev</id>
+             <name>Sergey Shelomentsev</name>
+             <email>sergey.v.shelomentsev@gmail.com</email>
+             <timezone>+7</timezone>
+         </developer>
+    </developers>
+
+    <scm>
+        <connection>scm:git:ssh://github.com:jmcgeheeiv/copy-to-slave-plugin.git</connection>
+        <developerConnection>scm:git:ssh://git@github.com/jmcgeheeiv/copy-to-slave-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/copy-to-slave-plugin</url>
+        <tag>copy-to-slave-1.4.7</tag>
+    </scm>
 
     <repositories>
         <repository>
             <id>repo.jenkins-ci.org</id>
-            <url>http://repo.jenkins-ci.org/public/</url>
+            <url>https://repo.jenkins-ci.org/public/</url>
         </repository>
     </repositories>
-
     <pluginRepositories>
         <pluginRepository>
             <id>repo.jenkins-ci.org</id>
-            <url>http://repo.jenkins-ci.org/public/</url>
+            <url>https://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
-    
-     <scm>
-      <connection>scm:git:git://github.com/jenkinsci/copy-to-slave-plugin.git</connection>
-      <developerConnection>scm:git:https://github.com/jenkinsci/copy-to-slave-plugin.git</developerConnection>
-      <url>https://github.com/jenkinsci/copy-to-slave-plugin</url>
-     </scm>
-     
-</project>  
-  
-
+    <issueManagement>
+        <url>https://github.com/jenkinsci/copy-to-slave-plugin/issues</url>
+        <system>GitHub</system>
```

Comment:
Issues for this plugin are currently managed on the Jenkins JIRA, not on GitHub; in fact, GitHub issues are disabled as of 2020-05-03